src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.UniversalDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    UniversalDao<Bet> betUniversalDao;
+    @Inject
+    UniversalDao<User> userUniversalDao;
+    private final Scanner scanner = new Scanner(System.in);
+
+    public void handle() {
+        handleUser();
+        handleBet();
+    }
+
+    private void handleBet() {
+        while (true) {
+            System.out.println("Please input your bet value and risk:");
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            String[] betData = command.split(" ");
+            checkInput(betData);
+            Bet bet;
+            try {
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
+                System.out.println("Bad input information!");
+                continue;
+            }
+            betUniversalDao.add(bet);
```

Comment:
I so this logic at Bogdan's video, but I really can't understand why do we add "null" to storage if there was "Bad input information!"? It's just question for discussion.

Reply:
We can't add a null bet because if we have a bet with the null or bad type fields we just go to the next iteration without adding to storage. What about the example from the video, I think that Bohdan makes it that way just for the demonstration and he corrected it in the refactor video.

Reply:
Oh, "continue" in catch, I see. It's steel not clear for me, why do we need add one extra line with "continue" to catch block instead jast moving add and sout to try block, but both variants will work correct of course.